boards/arduino/nicla_vision/camera_ext_clock.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 Felipe Neves
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/clock_control.h>
+
+#define LOG_LEVEL CONFIG_CLOCK_CONTROL_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(camera_ext_clock);
```

Comment:
I could be wrong, but I think this is the older logging declaration format, with a new format [introduced](https://github.com/zephyrproject-rtos/zephyr/commit/5e346812aca93570c544042c99122a059b35e48b) allowing `<zephyr/logging/log.h>` to be a normal header. you could go for the new style:

```suggestion
#include <zephyr/logging/log.h>

LOG_MODULE_REGISTER(camera_ext_clock, CONFIG_CLOCK_CONTROL_LOG_LEVEL);
```

Reply:
I think you are right, I picked this file from an old board init snippet.